pkg/kube/wait.go

```diff
@@ -153,7 +152,7 @@ func SelectorsForObject(object runtime.Object) (selector labels.Selector, err er
 	case *batchv1.Job:
 		selector, err = metav1.LabelSelectorAsSelector(t.Spec.Selector)
 	case *corev1.Service:
-		if t.Spec.Selector == nil || len(t.Spec.Selector) == 0 {
+		if len(t.Spec.Selector) == 0 {
```

Comment:
ditto comment on non-error related changes

(to be clear, I'm not against this change; just against combining it into this PR)